spec/services/migration/migrators/participant_id_change_spec.rb

```diff
@@ -28,8 +28,8 @@ def setup_failure_state
         participant_id_change = ParticipantIdChange.first
         expect(participant_id_change).to have_attributes({
           user_id: User.find_by(ecf_id: ecf_resource1.user_id).id,
-          from_participant_id: User.find_by(ecf_id: ecf_resource1.from_participant_id).id,
```

Comment:
could we add a test that checks if the from participant user doesn't exist in npq nothing fails?